tests/Controller/PageControllerTest.php

```diff
@@ -2,13 +2,45 @@
 
 namespace OCA\Analysis_App\Tests\Controller;
 
-use OCA\Analysis_App\Controller\PageController;
-use OCP\IRequest;
+use OCA\Analysis_app\Controller\PageController;
+use OCA\Analysis_app\FileModel;
+use OCP\AppFramework\Http\TemplateResponse;
 use Test\TestCase;
 
+
 class PageControllerTest extends TestCase {
-    public function testTrueIsTrue() {
-        $foo = true;
-        $this->assertTrue($foo);
+    /**
+     * @params $appName
+     * @params $fileModel
+     * @params $request
+     */
+    private $appName;
+    private $fileModel;
+    private $request;
+    private $expectedHtml;
+
+
+    protected function setUp() {
+        parent::setUp();
+        $this->appName = $this->createMock('\OCA\Analysis_app\Controller\PageController');
+        $this->request = $this->createMock('\OCP\IRequest');
+        $this->fileModel = $this->createMock('\OCA\Analysis_app\FileModel');
+        $this->expectedHtml = $this->getMockBuilder('OCP\AppFramework\Http\TemplateResponse')

```

Comment:
Expected HTML setUp in bir parçası değil. Sadece İndex metodunun bir parçası olabilir belki. Şöyle düşün bir controller birden fazla farklı response dönebilir. Bir de sadece class bağımlılıklarıni mock la. Expected HTML bir bağımlılık değil.